CHANGES.md

```diff
@@ -1,5 +1,22 @@
 # Change log for kotlinx.coroutines
 
+## Version 1.2.1
+
+Major:
+  * Infrastructure for testing coroutine-specific code in `kotlinx-coroutines-test`: `runBlockingTest`, `TestCoroutineScope` and `TestCoroutineDispatcher`, contributed by Sean McQuillan (@objcode).
```

Comment:
Should mention deprecation of obsolete test APIs from `kotlinx-coroutines-core` package.